src/app/components/address-displayer/form-address-displayer.tsx

```diff
@@ -15,7 +14,7 @@ export function FormAddressDisplayer({ address, ...rest }: FormAddressDisplayerP
       flexWrap="wrap"
       justifyContent="end"
       mr="-8px"
-      fontSize="16px"
+      fontSize="lg"
```

Comment:
Here we should probably use a `textStyle` instead?

Reply:
We should set `textStyle` on the `AddressDisplayerLayout` component a few levels below this

Reply:
Ok, will check this.